Assets/Mirror/Runtime/NetworkWriter.cs

```diff
@@ -40,7 +42,12 @@ public void SetLength(long value)
 
         public void Write(byte value) => writer.Write(value);
         public void Write(sbyte value) => writer.Write(value);
-        public void Write(char value) => writer.Write(value);
+        public void Write(char value)
+        {
+            // write char the same way that NetworkReader reads it (2 bytes)
+            byte[] bytes = BitConverter.GetBytes(value);
```

Comment:
This allocates,   just cast it to ushort and write that.   For reading read an ushort and cast it back to char.